bundles/core/src/main/java/com/adobe/cq/commerce/core/components/internal/services/ComponentsConfigurationAdapterFactory.java

```diff
@@ -0,0 +1,67 @@
+/*
+ *  Copyright 2020 Adobe. All rights reserved.
+ *
+ *   This file is licensed to you under the Apache License, Version 2.0 (the "License");
```

Comment:
Sorry to be picky, this header is not formatted like all the other headers we use.